palace/drivers/magnetostaticsolver.cpp

```diff
@@ -26,61 +26,61 @@ MagnetostaticSolver::Solve(const std::vector<std::unique_ptr<Mesh>> &mesh) const
   // handled eliminating the rows and columns of the system matrix for the corresponding
   // dofs.
   BlockTimer bt0(Timer::CONSTRUCT);
-  CurlCurlOperator curlcurlop(iodata, mesh);
-  auto K = curlcurlop.GetStiffnessMatrix();
-  const auto &Curl = curlcurlop.GetCurlMatrix();
-  SaveMetadata(curlcurlop.GetNDSpaces());
+  CurlCurlOperator curlcurl_op(iodata, mesh);
+  auto K = curlcurl_op.GetStiffnessMatrix();
+  const auto &Curl = curlcurl_op.GetCurlMatrix();
+  SaveMetadata(curlcurl_op.GetNDSpaces());
 
   // Set up the linear solver.
-  KspSolver ksp(iodata, curlcurlop.GetNDSpaces(), &curlcurlop.GetH1Spaces());
+  KspSolver ksp(iodata, curlcurl_op.GetNDSpaces(), &curlcurl_op.GetH1Spaces());
   ksp.SetOperators(*K, *K);
 
   // Terminal indices are the set of boundaries over which to compute the inductance matrix.
-  PostOperator postop(iodata, curlcurlop, "magnetostatic");
-  int nstep = static_cast<int>(curlcurlop.GetSurfaceCurrentOp().Size());
-  MFEM_VERIFY(nstep > 0,
+  PostOperator post_op(iodata, curlcurl_op, "magnetostatic");
+  int n_step = static_cast<int>(curlcurl_op.GetSurfaceCurrentOp().Size());
+  MFEM_VERIFY(n_step > 0,
               "No surface current boundaries specified for magnetostatic simulation!");
 
   // Source term and solution vector storage.
   Vector RHS(Curl.Width()), B(Curl.Height());
-  std::vector<Vector> A(nstep);
-  std::vector<double> I_inc(nstep);
+  std::vector<Vector> A(n_step);
+  std::vector<double> I_inc(n_step);
 
   // Initialize structures for storing and reducing the results of error estimation.
   CurlFluxErrorEstimator estimator(
-      curlcurlop.GetMaterialOp(), curlcurlop.GetRTSpace(), curlcurlop.GetNDSpaces(),
+      curlcurl_op.GetMaterialOp(), curlcurl_op.GetRTSpace(), curlcurl_op.GetNDSpaces(),
       iodata.solver.linear.estimator_tol, iodata.solver.linear.estimator_max_it, 0,
       iodata.solver.linear.estimator_mg);
   ErrorIndicator indicator;
 
   // Main loop over current source boundaries.
-  Mpi::Print("\nComputing magnetostatic fields for {:d} source boundar{}\n", nstep,
-             (nstep > 1) ? "ies" : "y");
+  Mpi::Print("\nComputing magnetostatic fields for {:d} source boundar{}\n", n_step,
+             (n_step > 1) ? "ies" : "y");
   int step = 0;
   auto t0 = Timer::Now();
-  for (const auto &[idx, data] : curlcurlop.GetSurfaceCurrentOp())
+  for (const auto &[idx, data] : curlcurl_op.GetSurfaceCurrentOp())
   {
-    Mpi::Print("\nIt {:d}/{:d}: Index = {:d} (elapsed time = {:.2e} s)\n", step + 1, nstep,
+    Mpi::Print("\nIt {:d}/{:d}: Index = {:d} (elapsed time = {:.2e} s)\n", step + 1, n_step,
                idx, Timer::Duration(Timer::Now() - t0).count());
 
     // Form and solve the linear system for a prescribed current on the specified source.
     Mpi::Print("\n");
     A[step].SetSize(RHS.Size());
     A[step].UseDevice(true);
     A[step] = 0.0;
-    curlcurlop.GetExcitationVector(idx, RHS);
+    curlcurl_op.GetExcitationVector(idx, RHS);
```

Comment:
Not `curl_curl_op`?

Reply:
I could go either way on this really but `curlcurl` seemed OK as a variable name. The big gain is separating `op` from the core subject.